SeleniumPracticalExercise/TestCases/AddEmployee.cs

```diff
@@ -1,31 +1,56 @@
 ﻿using NUnit.Framework;
+using SeleniumPracticalExercise.Common;
+using SeleniumPracticalExercise.PageObjects;
 using SeleniumPracticalExercise.TestCases.Common;
 
-namespace SeleniumPracticalExercise.TestCases
-{
-    class AddEmployee : BaseTestLocal
+class AddEmployee : BaseTestLocal
+
     {
         [Test]
         [Category("Add Employee")]
         public void AddEmployeeTest()
         {
+            string username = "Admin"; 
+            string password = "admin123"; 
+            string firstName = Utils.GenerateRandomString(6); 
+            string lastName = Utils.GenerateRandomString(8);
+
             // Steps to automate:
             // 1. Navigate to https://opensource-demo.orangehrmlive.com/web/index.php/auth/login
-            // 2. Log in using Username: Admin, Password: admin123
-            // 3. Click "PIM" in the left nav
+            Driver.Value.Navigate().GoToUrl("https://opensource-demo.orangehrmlive.com/web/index.php/auth/login");
+
+            // 2. Log in using Username: Admin, Password: admin123 using LoginPage - Page Object Model
+            LoginPage loginPage = new LoginPage(Driver.Value);
```

Comment:
Make this a one-liner... remember we talked about this last Thu? If the `loginPage` variable (or any variable for that matter) isn't used more than once, we generally make it a one-liner. Having said that, if that one line becomes excessively long then two lines are fine. Readability should always trump fewer lines.

Reply:
made it a one-liner
new LoginPage(Driver.Value).Login(username, password);